### PR TITLE
Added URL field

### PR DESCRIPTION
**Pull request by assertchris**

### CODE REVIEW

File: _config/config.yml [hunk 1]
```diff
@@ -10,3 +10,4 @@ Controller:
 
 QuickFeedbackExtension:
   rate_limit: 3
+  redirect_field: false
```
Comment (camfindlay) [inline _config/config.yml]: No need for this already set in the class private static.

File: code/extensions/QuickFeedbackExtension.php [hunk 2]
```diff
@@ -19,17 +26,25 @@ class QuickFeedbackExtension extends DataExtension {
...
+			ButtonGroupField::create('Rating', '', array(
+				'1' => _t('QuickFeedback.Yes', 'Yes'),
+				'0' => _t('QuickFeedback.No', 'No'),
+			)),
+			TextareaField::create('Comment', _t('QuickFeedback.Comment', 'Comment'))
+		);
+
+		if ((bool) Config::inst()->get('QuickFeedbackExtension', 'redirect_field')) {
+			$fields->push(
+				HiddenField::create('Redirect', null, $_SERVER['REQUEST_URI'])
```
Comment (camfindlay) [inline code/extensions/QuickFeedbackExtension.php:40]: Any risk of XSS here? Accepting user input and then using to redirect?

File: code/extensions/QuickFeedbackExtension.php [hunk 3]
```diff
@@ -82,19 +103,31 @@ public function doSubmit($data, $form) {
...
+
+		if ((bool) Config::inst()->get('QuickFeedbackExtension', 'redirect_field') && isset($data['Redirect'])) {
+			$feedback->URL = $data['Redirect'];
+		}
+
 		$feedback->write();
 
 		$form->sessionMessage(_t('QuickFeedback.ThanksMessage', 'Thanks for your comment!'),'good');
-		return $this->owner->redirect(Director::baseURL() . $this->owner->URLSegment . '?success=1');
+		return $this->owner->redirect($default_redirect . '?success=1');
```
Comment (camfindlay) [inline code/extensions/QuickFeedbackExtension.php]: See above

Comment (camfindlay) [inline code/extensions/QuickFeedbackExtension.php]: Does this also jump the user down the page to the message?

File: _config/config.yml [hunk 4]
```diff
@@ -10,3 +10,4 @@ Controller:
 
 QuickFeedbackExtension:
   rate_limit: 3
```
Comment (camfindlay) [inline _config/config.yml]: No need for this either (though this made it's way in from another PR).

File: _config/config.yml (hunk 1 above)
Comment (assertchris) [inline _config/config.yml]: I wanted to show the configuration parameter. The class property serves as a default, if this config parameter is removed.

File: code/extensions/QuickFeedbackExtension.php (hunk 2 above)
Comment (assertchris) [inline code/extensions/QuickFeedbackExtension.php:40]: See elsewhere.

**Pull request merged**